src/components/Course/CourseTextArea.jsx

```diff
@@ -0,0 +1,13 @@
+import { Hero , Textarea} from "react-daisyui";
+
+
+
+function CourseTextArea() {
+  return(
+    <Textarea>
```

Comment:
Dont get it lesson markdown or lesson navbar?

Reply:
this was just for a template.

Reply:
to determine how we will input everything from the class data

Reply:
Okey lets just keep it for now

Reply:
Agreed?

Reply:
agreed